src/server/debug.rs

```diff
@@ -332,7 +332,7 @@ impl Debugger {
         Ok(res)
     }
 
-    pub fn remove_failed_stores(&self, store_ids: Vec<u64>) -> Result<()> {
+    pub fn remove_failed_stores(&self, store_ids: Vec<u64>, region_ids: Vec<u64>) -> Result<()> {
```

Comment:
This function seems to have the assumption that `region_ids.len() == 0` means that the user intends to cover all regions. I worry that this will not always be the case. Worse, future users may incorrectly assume they need to pass **all** region ids to this function.

Could you either note this assumption in a documentation comment or (preferably) make the function accept a `region_ids: Option<Vec<u64>>` instead? Where `None` would communicate clearly that no limited set is specified.